FileManager/FileManager/ViewController.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  ViewController.swift
+//  FileManager
+//
+//  Created by Abdullah Alnutayfi on 23/11/2021.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    var folders : [String] = []
+    var selectedFolder = ""
+  
+    let textField = UITextField()
+    let tableView = UITableView()
+    let textView = UITextField()
+    let button1 = UIButton()
+    let button2 = UIButton()
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        fetchData()
+        setup()
+    }
+
+    func setup(){
```

Comment:
maybe its better to rename this function to something like `settingUpUI`, `setup` here does't explain the purpose